.github/workflows/ci.yml

```diff
@@ -8,44 +8,38 @@ env:
 
 jobs:
   checkers:
+    name: Run static checkers
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - name: Run clang-format style check (.c and .h)
       uses: jidicula/clang-format-action@master
 
-  build:
+  sanitizers:
```

Comment:
I guess this should be named "build", now that "build" and "sanitizers" are unified, right?

Reply:
right!